### ISSUE TITLE
[BUG] Re-Publish own kata after making any changes doesn't work

### ISSUE THREAD

**Issue by Dentzil** [Codewars/codewars.com#826]
I can not re-publish [my new kata](https://www.codewars.com/kata/extension-groupwhile-consecutive-by-the-condition-items/csharp) after making any changes in Test Cases.

The shown error message says "_Final Solution is invalid. (csharp) The solution should pass the provided test cases_", thought Validate Solution is always successful.
<img width="534" alt="bildschirmfoto 2017-02-12 um 12 29 07" src="https://cloud.githubusercontent.com/assets/12846346/22861675/232dddc4-f11f-11e6-8eae-97f29f12d53e.png">

And it's not matter what changes were made. Even the adding some new Console Output fails.

**Comment by g964** [comment]
@jhoffner 
Same thing for that kata:
<https://www.codewars.com/kata/569218bc919ccba77000000b/edit/clojure>
thought Validate Solution is always successful!

**Comment by jhoffner** [comment]
I had fixed an issue that was causing some challenges to be published even though they were actually failing. Specifically cases where all of the tests passed but where the program had stderr output or unclean program exits. 

@Dentzil I just tried republishing your challenge and it works. Not sure what changes you had made. 

@g964 For your challenge it looks like both Haskell and Clojure are both failing when I validate. Clojure has some STDERR warnings (I'm going to add it to my backlog to actively try to ignore warnings) and haskell is failing completely.

**Comment by g964** [comment]
@jhoffner
Clojure: the warnings are there because being unable to re-publish with a small modification of the name of a function in the tests I had to put a small function in the preloaded part. Furthermore when I "VALIDATE" only "Example Test Cases" are run. 

Clojure seems to have problem elsewhere:
in this kata (I wanted to modify it)
<https://www.codewars.com/kata/56efab15740d301ab40002ee/edit/clojure>
when I run the tests their order is completely mixed. Example:
```
Random Tests som
✔ Test Passed
      a-test3
          Basic test 3 : gcdi lcmu som mini maxi
             ✔ Test Passed
             Basic test 3 : gcdi lcmu som mini maxi
             Basic test 3 : gcdi lcmu som mini maxi
            ✔ Test Passed
            a-test5
                Random Tests gcdi
               ✔ Test Passed
               Random Tests gcdi
```
I tried on "Preview". I got a server error 500 and
`Cannot allocate memory - nodejs (in /codewars/vendor/assets/javascripts/intro.js)`

Haskell: It worked previously...

Is it possible to delete a translation of a kata in a given language (small cross at the right of the language; is it really effective)? I am asking because some time ago I had problem with that.

Wow! Painfull:-)
